project1/styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+    background-color: rgb(255, 220, 249);
+    font-family: 'Cute Font', cursive;
+    font-family: 'Lato', sans-serif;
+    font-family: 'Roboto', sans-serif;
+
+}
+
+* {
+    box-sizing: border-box;
+}
+
+main {
+    /* border: 10px solid #444; */
+
+    width: 1310px;
+    padding: 70px;
+    border: 20px solid rgb(255, 199, 232);
+    background-color: rgb(253, 171, 210);
+    text-align: left;
+    margin: 0 auto;
+}
+
+h1 {
```

Comment:
Look for places where you can make it clear what styles are shared between similar elements. This makes your CSS clearer and allows you to write fewer rules. So, I'd write:

```css
h1, h2, h3 {
    font-family: 'Prata', serif;
}
```
and delete the individual `font-family` declarations below.